ext/app/electron/logins.ts

```diff
@@ -1,22 +1,29 @@
 import {ApiError} from 'app/common/ApiError';
+import { UserProfile } from 'app/common/LoginSessionAPI';
```

Comment:
please try to keep import style consistent with surroundings.

Reply:
Changed all imports to have surrounding spaces in braces, double quotes, and eslint's recommended ordering.